docstrfmt/docstrfmt.py

```diff
@@ -837,7 +837,14 @@ def literal(
     def literal_block(
         self, node: docutils.nodes.literal_block, context: FormatContext
     ) -> line_iterator:
-        yield "::"
+        if (
+            len(node.attributes["classes"]) > 1
+            and node.attributes["classes"][0] == "code"
```

Comment:
This I think will need to be `code` and `code-block`.

```suggestion
            and node.attributes["classes"][0] in ["code", "code-block"]
```